examples/gradle/multi-module/build.gradle

```diff
@@ -43,10 +44,28 @@ allprojects {
     }
 
     dependencies {
-        compile 'org.scala-lang:scala-library:2.12.6'
+        compile 'org.scala-lang:scala-library:2.12.8'
         scoverage 'org.scoverage:scalac-scoverage-plugin_2.12:1.3.1',
                 'org.scoverage:scalac-scoverage-runtime_2.12:1.3.1'
         testCompile 'org.scalatest:scalatest_2.12:3.0.5'
         testRuntime 'org.pegdown:pegdown:1.4.2'
     }
+
+    // Scapegoat configuration.
+    configurations {
+        scalaCompilerPlugin
+    }
+
+    dependencies {
+        compile 'com.sksamuel.scapegoat:scalac-scapegoat-plugin_2.12:1.3.8' // to prevent https://github.com/sksamuel/scapegoat/issues/98
+        scalaCompilerPlugin "com.sksamuel.scapegoat:scalac-scapegoat-plugin_2.12:1.3.8"
+    }
+
+    tasks.withType(ScalaCompile) {
```

Comment:
So this executes scapegoat when compiling the project? No need to run an additional task like with sbt?

Reply:
Exactly. The same applies with Maven.
It is somewhat convenient. But, I personally prefer the separation of concerns that the SBT plugin brings.